packages/cli/src/net/createFetch.mts

```diff
@@ -61,6 +61,8 @@ function handleFetchError(e: unknown): Promise<Response> {
     tip = "Check your token is valid and has not expired or been disabled";
   } else if (e.statusCode === 403) {
     tip = "Check your token has the required scopes for this operation";
+  } else if (e.errorName === "Artifacts:ArtifactAlreadyExists") {
+    e.message = "Version already exists";
```

Comment:
Would prefer to not mutate the error (even though it's not passed on) and instead:

```
let message = e.message;
if (..) {
  ...
}
```

this means that we also keep the two branches for status code and error name separate so if it happens that they overlap they still both run on the message and tip